docs/reference/eql/eql-search-api.asciidoc

```diff
@@ -160,6 +160,18 @@ Defaults to `event.category`, as defined in the {ecs-ref}/ecs-event.html[Elastic
 Common Schema (ECS)]. If an index does not contain the `event.category` field,
 this value is required.
 
+`fetch_size`::
```

Comment:
Currently, this value has to be larger than 2. Note sure though whether it makes sense mentioning this.

Reply:
I would add that this value cannot be higher `index_max_result_window` (one can try but will end up with an exception from ES).